reference/employer/openapi.yaml

```diff
@@ -1,7 +1,7 @@
 openapi: 3.0.0
 info:
   title: API Reference
-  version: '2020-09-17'
+  version: "2020-09-17"
```

Comment:
For my understanding, why were these changes from single to double quotes? I don't know much about OpenAPI yet. Is this file auto-generated?

Reply:
Honestly, no clue. It might have been an auto-save correction by a yaml linter. Looking through the rest of the document, it looks like double quotes is used everywhere else.

Reply:
Yea I think this PR updated all the single quotes to double quotes. I'd pull that change out of this PR just to keep the diff focused on the relevant docs changes. If you think these all _should_ be double quotes, I can get behind that but I think it should be in a separate change set.